experimental/uefi/app/.cargo/config.toml

```diff
@@ -2,7 +2,7 @@
 target = "x86_64-unknown-uefi"
 
 [target.x86_64-unknown-uefi]
-runner = "qemu-system-x86_64 -nodefaults -nographic -bios /usr/share/OVMF/OVMF_CODE.fd -serial file:target/console.log -serial stdio -machine q35 -device isa-debug-exit,iobase=0xf4,iosize=0x04 -kernel"
+runner = "qemu-system-x86_64 -enable-kvm -cpu Broadwell-IBRS -nodefaults -nographic -bios /usr/share/OVMF/OVMF_CODE.fd -serial file:target/console.log -serial stdio -machine q35 -device isa-debug-exit,iobase=0xf4,iosize=0x04 -kernel"
```

Comment:
When reviewing keep in mind that this will require all of us to have KVM on the host enabled in order to use the runner. 

Imo that's preferable to having multiple runners using different settings, but worth pointing out. 

We also cannot easily set a different runner based on eg a KVM feature gate. This is since the while `cfg` is technically available in the config, the values available to it are very limited. Ref: https://github.com/project-oak/oak/pull/2725